_posts/2017/2017-02-23-why-i-love-working-as-a-scrum-master.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Why I love working as a Scrum Master"
+date: "2017-02-23 07:00:00"
+categories: working-at-epages
```

Comment:
@BirgitBader I'm waiting for Ann-Katrin to be ready and then take a picture of her